File: single-language-analysis/dataset-files/python/test_smell_087.py
# Original URL: https://github.com/entrymissing/advent-of-code-23/blob/257265de25a632b39c1d4ef23f7c94c1b5fea3ac/day17.py#L155-L158

def test_smell_087():
  if os.path.exists('input'):
    assert solve_1() == 102
    assert solve_2() == 94

File: single-language-analysis/dataset-files/CSharp/CSharp.Tests/Tests/TestSmell087.cs
using Xunit;
using System.IO;

public class TestSmell087
{
    [Fact]
    public void TestResultsMethod()
    {
        if (File.Exists("input"))
        {
            Assert.Equal(102, Solve1());
            Assert.Equal(94, Solve2());
        }
    }

    private int Solve1()
    {
        // Implementation of Solve1
        return 0;
    }

    private int Solve2()
    {
        // Implementation of Solve2
        return 0;
    }
}